common.blocks/button/_type/button_type_link.bh.js

```diff
@@ -6,7 +6,7 @@ module.exports = function(bh) {
         json.target && ctx.attr('target', json.target);
         ctx.mod('disabled')?
             ctx
-                .attr('aria-disabled', true)
+                .attr('aria-disabled', 'true')
```

Comment:
In 4.1, https://github.com/bem/bh/compare/v4.0.0...v4.1.0

Boolean attributes support.